### ISSUE TITLE
Ability to turn off warnings

### ISSUE THREAD

**Issue by hworld** [Polyconseil/vue-gettext#11]
Would be great to have a way to turn off warnings of untranslated string. Basically this line:

```
console.warn(`Untranslated ${language} key found:\n${msgid}`)
```

It is helpful for development, although it can get a bit noisy when adding new code without translations yet. Also, in production it'd be nice not to show as translations on my site are done by the community and many aren't translated yet.

**Comment by kemar** [comment]
Yes, I understand this can be annoying.

I think I'll add a new `silent` config option which will have the same value as [`Vue.config.silent`](https://vuejs.org/v2/api/#silent) by default.

**Comment by kemar** [comment]
It's available in [v2.0.8](https://github.com/Polyconseil/vue-gettext/releases/tag/v2.0.8).
